.github/labeler-modules.yml

```diff
@@ -0,0 +1,62 @@
+scipy.cluster:
```

Comment:
I would merge the two files no? Otherwise my understanding is that it would create 2 jobs, hence cost more CI which we want to limit.

Reply:
There should be no issue with combining them. The reason `scikit-learn` did not combine our jobs was that `max-labels` was configured differently for the modules.

XREF: https://github.com/scikit-learn/scikit-learn/blob/335e069acd7bb8baac654b69a7995a3d87275c94/.github/workflows/labeler-module.yml#L15

Reply:
Indeed. The files are merged by 370d13e2bf9b677b91536255dfbbb7ef4186fac5.